test/unit_test/rag/llm/test_embedding_model.py

```
@@ -82,6 +85,26 @@ def _make_openai(cls=OpenAIEmbed, total_tokens=None):
     return embed
 
 
+@pytest.mark.p1
+@pytest.mark.parametrize("embed_cls", [TogetherAIEmbed, OpenAI_APIEmbed])
+def test_openai_compatible_embedding_does_not_send_litellm_drop_params(embed_cls):
+    """Strict OpenAI-compatible APIs reject LiteLLM-only request fields."""
+
+    def _strict_create(input, model, **kwargs):
```

Ruff (0.16.1)

[error] Function argument `input` is shadowing a Python builtin

(A002)